src/test/java/com/coherentsolutions/training/aqa/java/web/nikonova/objects/BaseTest.java

```diff
@@ -0,0 +1,55 @@
+package com.coherentsolutions.training.aqa.java.web.nikonova.objects;
+
+import com.coherentsolutions.training.aqa.java.web.nikonova.browsers.BrowsersProperties;
+import com.coherentsolutions.training.aqa.java.web.nikonova.browsers.BrowsersSettings;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.WebDriver;
+
+import org.testng.annotations.*;
+
+import java.time.Duration;
+
+public class BaseTest {
+
+    private static final String PATH_BROWSER = "./src/main/resources/drivers/chromedriver.exe";
+
+    protected static final String URL = "http://automationpractice.com/index.php?controller=authentication&back=my-account";
+
+    protected WebDriver driver;
+
+    public static Logger log = LogManager.getLogger();
+
+    @BeforeSuite
+    public void setUp() {
+        if (driver == null) {
+            log.info("Tests are starting!");
+        }
+    }
+
+    @AfterSuite
+    public void tearDown() {
+        log.info("Tests are ending!");
+        if (driver != null) {
+            driver.quit();
+        }
```

Comment:
dead code. Driver will be always == null, because we have  this method:
@AfterClass
    public void cleanup() {
        BrowsersSettings.driverTeardown();
    }